### PR TITLE
[ENHANCEMENT] Add doc + renaming row to panel group

### PR DESCRIPTION
**Pull request by Gladorme**
# Description

Last details for Go SDK with more doc

# Checklist

- [x] Pull request has a descriptive title and context useful to a reviewer.
- [x] Pull request title follows the `[<catalog_entry>] <commit message>` naming convention using one of the
  following `catalog_entry` values: `FEATURE`, `ENHANCEMENT`, `BUGFIX`, `BREAKINGCHANGE`, `DOC`,`IGNORE`.
- [x] All commits have [DCO signoffs](https://github.com/probot/dco#how-it-works).

### CODE REVIEW

**Comment by Nexucis** [comment]
nice ! Can you add an example on each docs please ? Thanks

File: docs/dac/go/prometheus/query.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Prometheus Datasource Builder
```
Comment (AntoineThebaud) [inline docs/dac/go/prometheus/query.md:1]:
```suggestion
# Prometheus Query Builder
```

**Review by AntoineThebaud — APPROVED** [review]
LGTM! just spotted one typo

**Review by Nexucis — APPROVED** [review]
That's awesome !! Thank you !

**Pull request merged**